install.sh

```diff
@@ -750,7 +750,7 @@ case "$SHELL" in
 esac
 if [[ "$UNAME_MACHINE" == "arm64" ]] || [[ -n "${HOMEBREW_ON_LINUX-}" ]]; then
   cat <<EOS
-- Add Homebrew to your ${tty_bold}PATH${tty_reset} in ${tty_underline}${shell_profile}${tty_reset}:
+- Run these two commands to add Homebrew to your ${tty_bold}PATH${tty_reset} via ${tty_underline}${shell_profile}${tty_reset}:
```

Comment:
PATH via profile -> PATH in profile via

Reply:
Hm, what about just removing the `${shell_profile}` part entirely? I feel like including that in any way might still confuse some people.

Reply:
@alebcay sounds good to me. We can also make it
```suggestion
- Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
```

Reply:
Same modification at line 763:

https://github.com/Homebrew/install/blob/61f40dfdd5f904122f7cfbc2d4c91b31290d4608/install.sh#L763-L765

Reply:
How about just `Run these commands to complete your Homebrew installation`?

Reply:
Good catch for additional modification and thanks also for that wording suggestion. Adding both of those now.

Reply:
@XuehaiPan for that additional modification, is it a constant number of commands (e.g. 2)? Or variable number? I'm not as familiar with the syntax used here.

Reply:
@gromgit I think I draw the line at "being aware that `PATH` is a thing used in the terminal" as the minimum level of literacy here 😄

Reply:
> is it a constant number of commands (e.g. 2)
It's 2 (for set homebrew/brew or homebrew/brew) or 4 (for set both).

Reply:
> - Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:

I like this 👍🏻